### ISSUE TITLE
Model divege after loading from a pretain-model,but the result of evluating is at right aac on train set.

### ISSUE THREAD

**Issue by CXOldStar** [keras-team/keras#11688]
## Hello, I meet a problem. 
When I want to load a pretrain model,that has been trained to get 90% aac on training dataset,  and then train it more epochs, the model start diverge and the aac of train dataset toward 0.Then I use the model.evaluate_generator() to evluate the train dataset with same others code.The evluate_generator() return a 90% aac. So the weights is loaded all right and model should not diverge if I train it more steps.
### Info:
The method,used to save model, is keras.callback.ModelCheckpoint().
The optimize is SGD.
Version:
python: 3.5.2
keras:2.2.4
tensorflow-gpu:1.10.1 
### The methods I have tried:
1.Load weigths
```
resnet_x = ResNet34(input_tensor, input_shape=feature_shape, classes=classes, include_softmax=True)
model = Model(inputs=input_tensor, outputs=resnet_x, name='resnet34')
tf.global_variables_initializer()
model.load_weights('{}/{}'.format(save_path, weights_filename))
# Then compile it with SGD
# Then training it by fit_generator()
```
2.Directly load model 
```
model = load_model(file_name)
# Then training it by fit_generator()
```
3.set the learning rate to very small in SGD.
```
# lr=0.01 when I train model first time.
sgd = SGD(lr=0.00001, decay=1e-4, momentum=0.9, nesterov=True)
```
All of them do not work for me and show the similar line to me:
```
428/4266 [==>...........................] - ETA: 5:07:54 - loss: 16.0567 - acc: 0.0027
```
Please help me to solve it.
Best regards!
